common-use-cases/css-transformations.md

```diff
@@ -39,4 +39,4 @@ You can apply the transformations on the parent object of a chart.
 ```
 
 Here is an exmaple (refer to the HTML tab to see the source code):
-<iframe width="100%" height="600" src="//jsfiddle.net/fusioncharts/hzc1rLxa/embedded/result,html,js/" allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"></iframe>
\ No newline at end of file
+<iframe width="100%" height="600" src="//jsfiddle.net/fusioncharts/hzc1rLxa/4/embedded/result,html,js/" allowfullscreen="allowfullscreen" allowpaymentrequest frameborder="0"></iframe>
```

Comment:
I had updated the base version of the main fiddle itself hence it was not needed to provide the version. If tomorrow, we change the fiddle again we don't have to make the changes here. That was the idea. But it seems OK. Not changing it back.